src/theory/strings/core_solver.cpp

```diff
@@ -529,17 +529,25 @@ void CoreSolver::checkNormalFormsEq()
   std::map<Node, Node> nf_to_eqc;
   std::map<Node, Node> eqc_to_nf;
   std::map<Node, Node> eqc_to_exp;
+  std::vector<CoreInferInfo> pinfer;
```

Comment:
I know none of these local variables is documented, but perhaps at least this one can be?

Reply:
Now documented.